Building 8.2.9 wheel on macOS fails due to missing libdeflate

I'm having troubles installing asammdf from source on my Intel Mac - when pip tries to build the wheel it fails. If I clone the asammdf repo and also the submodule for libdeflate I can build the wheel without problems. Any ideas what to do here, can pip somehow also clone the submodule?

```
➜ pip install asammdf==8.2.9                                                                                                                                                                                                                                     
Collecting asammdf==8.2.9
  Using cached asammdf-8.2.9.tar.gz (8.6 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: canmatrix>=1.2 in ./venv/lib/python3.12/site-packages (from canmatrix[arxml,dbc]>=1.2->asammdf==8.2.9) (1.2)
Requirement already satisfied: isal in ./venv/lib/python3.12/site-packages (from asammdf==8.2.9) (1.7.2)
Requirement already satisfied: lxml>=4.9.3 in ./venv/lib/python3.12/site-packages (from asammdf==8.2.9) (5.3.2)
Requirement already satisfied: lz4 in ./venv/lib/python3.12/site-packages (from asammdf==8.2.9) (4.4.4)
Requirement already satisfied: numexpr>=2.10.0 in ./venv/lib/python3.12/site-packages (from asammdf==8.2.9) (2.10.2)
Collecting numpy>=2.0.0 (from asammdf==8.2.9)
  Using cached numpy-2.2.4-cp312-cp312-macosx_14_0_x86_64.whl.metadata (62 kB)
Requirement already satisfied: pandas>=2.2.2 in ./venv/lib/python3.12/site-packages (from asammdf==8.2.9) (2.2.3)
Requirement already satisfied: python-dateutil in ./venv/lib/python3.12/site-packages (from asammdf==8.2.9) (2.9.0.post0)
Requirement already satisfied: typing-extensions~=4.10 in ./venv/lib/python3.12/site-packages (from asammdf==8.2.9) (4.13.1)
Requirement already satisfied: attrs>=19.2.0 in ./venv/lib/python3.12/site-packages (from canmatrix>=1.2->canmatrix[arxml,dbc]>=1.2->asammdf==8.2.9) (25.3.0)
Requirement already satisfied: click in ./venv/lib/python3.12/site-packages (from canmatrix>=1.2->canmatrix[arxml,dbc]>=1.2->asammdf==8.2.9) (8.1.8)
Requirement already satisfied: pytz>=2020.1 in ./venv/lib/python3.12/site-packages (from pandas>=2.2.2->asammdf==8.2.9) (2025.2)
Requirement already satisfied: tzdata>=2022.7 in ./venv/lib/python3.12/site-packages (from pandas>=2.2.2->asammdf==8.2.9) (2025.2)
Requirement already satisfied: six>=1.5 in ./venv/lib/python3.12/site-packages (from python-dateutil->asammdf==8.2.9) (1.17.0)
Using cached numpy-2.2.4-cp312-cp312-macosx_14_0_x86_64.whl (6.7 MB)
Building wheels for collected packages: asammdf
  Building wheel for asammdf (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for asammdf (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
      *** scikit-build-core 0.11.1 using CMake 4.0.0 (wheel)
      *** Configuring CMake...
      loading initial cache file /var/folders/j7/bcbs6m61771dbd41wgwrq_dw0000gn/T/tmpj90ttqii/build/CMakeInit.txt
      -- The C compiler identification is AppleClang 16.0.0.16000026
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /usr/bin/clang - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Found Python: venv/bin/python3.12 (found version "3.12.8") found components: Interpreter Development.Module Development.SABIModule NumPy
      CMake Error at CMakeLists.txt:20 (add_subdirectory):
        add_subdirectory given source "ext/libdeflate" which is not an existing
        directory.


      -- Configuring incomplete, errors occurred!

      *** CMake configuration failed
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for asammdf
Failed to build asammdf
ERROR: Failed to build installable wheels for some pyproject.toml based projects (asammdf)
```

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, ubuntu-24.04-arm, macos-latest, windows-latest]
+        os: [ubuntu-latest, ubuntu-24.04-arm, macos-13, macos-latest, windows-latest]
       fail-fast: false
 
     steps:

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -76,6 +76,7 @@ Source = "https://github.com/danielhrisca/asammdf"
 [tool.scikit-build]
 metadata.version.provider = "scikit_build_core.metadata.regex"
 metadata.version.input = "src/asammdf/version.py"
+sdist.include = ["ext"]
 wheel.packages = ["src/asammdf"]
 wheel.py-api = "cp310"
 

